### PR TITLE
Wall

### PR DESCRIPTION
**Pull request by Sindex42**
Add Pylint to Travis CI, this means branches need to be linted before merging

### CODE REVIEW

File: src/constants.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
+''' Colours and Window constants '''
+
 # colors
 GREEN = (14, 81, 53)
 GOLD = (212, 206, 70)
 DARK_GREY = (73, 75, 75)
 DARK_LINE = (23, 25, 25)
 RED = (124, 10, 2)
+BROWN = (59, 15, 15)
```
Comment (CKKH) [inline src/constants.py:9]: nice

File: src/main.py [hunk 2]
```diff
@@ -1,61 +1,77 @@
...
-
-        for event in pg.event.get():
-            if event.type == pg.QUIT:
-                self.playing = False
-
-
-# create instance of game
-GAME = Game()
-GAME.new()
+''' Main module for running pg '''
```
Comment (CKKH) [inline src/main.py:1]: Nice work making this read easily

File: src/main.py [hunk 3]
```diff
@@ -1,61 +1,77 @@
...
+        self.map_data = []
+        self.all_sprites = pg.sprite.Group()
+
+        self.load_data()
+
+    def new(self):
+        ''' Creates sprites '''
+
+        self.all_sprites.add(Enemy(1, 1))
+        for row, tiles in enumerate(self.map_data):
```
Comment (CKKH) [inline src/main.py:29]: Awesome implementation of walls

File: src/map.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+1111111111111111
```
Comment (CKKH) [inline src/map.txt:1]: 10101001110101

**Review by CKKH — APPROVED** [review]
Great work, so close to MVP!

**Review by hotblack86 — APPROVED** [review]
Good work on implementing the wall maps. Game nicely refactored too.

**Review by matharotheelf — COMMENTED** [review]
Well done

**Review by matharotheelf — APPROVED** [review]
Well done guys

**Pull request merged**